iaso/tests/api/test_serializers.py

```diff
@@ -251,10 +254,12 @@ def test_small_search(self):
         self.assertDictEqual(
             res,
             {
+                "creator": None,
                 "id": orgunit.id,
                 "name": "Endor Jedi Squad 1",
                 "org_unit_type_name": "Jedi Squad",
                 "parent": {
+                    "creator": None,
```

Comment:
Maybe we can improve the tests (or add a new one), to make sure the creator field has the expected value in case it's not None? thanks!

Reply:
Yep, done